### PR TITLE
A program that counts words split from sentences

### PR DESCRIPTION
**Pull request by ghost**
- Meant to show users how easy this can be done on concord
  compared to storm

### CODE REVIEW

File: concord_storm_wc/sentence_generator.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+require 'concord'
+
+def log(str)
```
Comment (ghost) [inline concord_storm_wc/sentence_generator.rb]: This method is boilerplate code, maybe i can put it inside of concord-ruby/utils.rb?

Comment (emaxerrno) [inline concord_storm_wc/sentence_generator.rb]: we should name it properly then

`log_to_stderr`

File: concord_storm_wc/word_counter.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+require 'concord'
+
+def log(str)
```
Comment (emaxerrno) [inline concord_storm_wc/word_counter.rb]: :+1: for cleaning up.

File: concord_storm_wc/sentence_splitter.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    words.each do |word|
+      # .. send word as key to downstream operators
+      context.produce_record('words', word, '')
+    end
+  end
+
+  def metadata
+    Concord::Utils.log_to_stderr("Metadata called")
+    Concord::Metadata.new(name: 'sentence-splitter',
+                          istreams: ['sentences'],
```
Comment (emaxerrno) [inline concord_storm_wc/sentence_splitter.rb]: Use ROUND_ROBIN strategy.

**Comment by emaxerrno** [comment]
LGTM

**Pull request merged**